typing/solver.ml

```diff
@@ -831,75 +844,81 @@ module Solver_mono (H : Hint) (C : Lattices_mono) = struct
       right_hint : ('a, right_only) hint_raw
     }
 
-  let submode (type a r l) (obj : a C.obj) (a : (a, allowed * r) mode)
-      (b : (a, l * allowed) mode) ~log =
-    let submode_cc ~log:_ obj left left_hint right right_hint =
+  let submode (type a r l) (pp : H.Pinpoint.t) (obj : a C.obj)
```

Comment:
Where does `submode` actually use the pinpoint? I see how it keep track of it through left/right adjoints, but I think I'm missing the actual usage site.